mxd/3.2_MXD_tutorial2_Access_policies.md

```diff
@@ -0,0 +1,129 @@
+# Restricting users from accessing an asset
+
+Bob will once again be the data provider and Alice is interested in Bob’s data assets. Bob, as a data provider, creates an asset. The data asset should have the following properties:
+
+| ID          | 4                                                                                            |
```

Comment:
this table confuses rather than it explains, since the same information is contained in the JSON example further down. pls remove.